extension/page-detect.js

```diff
@@ -33,6 +33,8 @@ window.pageDetect = (() => {
 
 	const isBlame = () => isRepo() && /^\/blame\//.test(getRepoPath());
 
+	const isSingleFile = () => isRepo() && /\/blob\//.test(getRepoPath());
```

Comment:
This will trigger a false-positive in the scenario that a user is viewing the `tree` and is looking inside a folder nested inside another named `blob`. I think there are 2 options to fix this:

1. Move the `repoUrl` var from `content.js` >> `page-detect.js`, and look for `{user}/{repo}/blob/` in the regex
2. Check the DOM for an element that only exists on the target page

Anyone have a strong preference? Leaning towards 1 since it'll make future unit-testing slightly easier without the need for DOM